explainer.md

```diff
@@ -99,19 +99,20 @@ async function OnVRAvailable() {
   // has that capability the page will want to add an "Enter VR" button (similar
   // to "Enter Fullscreen") that triggers the page to begin showing imagery on
   // the headset.
-  let exclusiveMode = await vrDevice.supportsSession({ exclusive: true });
-  if (exclusiveMode) {
+  vrDevice.supportsSession({ exclusive: true }).then(() => {
     var enterVrBtn = document.createElement("button");
     enterVrBtn.innerHTML = "Enter VR";
     enterVrBtn.addEventListener("click", BeginVRSession);
     document.body.appendChild(enterVrBtn);
-  }
+  }).catch(() => {
+    console.log("VRDevice does not support exclusive sessions.");
```

Comment:
Nit: Perhaps we want to word this error message a bit differently in case we add reasons for rejection that aren't simply because of hardware capability.  For example, in the past we've talked about the idea of the user being able to "disable VR" via some setting in the UA?